test/checkers/trace_checker.c

```diff
@@ -131,15 +186,16 @@ coldtrace_writer_close(void *page, const size_t size, uint64_t tid)
         if (!next->set)
             log_fatal("thread=%lu entry=%d found=%s but trace empty", tid, i,
                       coldtrace_entry_type_str(type));
-
-        if ((type != next->type))
-            log_fatal("thread=%lu entry=%d found=%s expected=%s", tid, i,
-                      coldtrace_entry_type_str(type),
-                      coldtrace_entry_type_str(next->type));
-
-        log_info("thread=%lu entry=%d match=%s", tid, i,
-                 coldtrace_entry_type_str(next->type));
-        next++;
+        // //no match
```

Comment:
I think that you forgot these comments here. If you wan to disable them most of the time, use some `#ifdef`.